controllers/v1/assessmentsController.js

```diff
@@ -277,6 +277,7 @@ module.exports = class Assessments {
                     frameworkExternalId: solutionDocument.frameworkExternalId,
                     entityTypeId: solutionDocument.entityTypeId,
                     entityType: solutionDocument.entityType,
+                    scoringSystem : solutionDocument.scoringSystem,
```

Comment:
@deepakr123 - Add isRubricDriven key in the submission document.

Reply:
resolved